Content.Server/Damage/Components/DamageOnLandComponent.cs

```diff
@@ -11,8 +11,8 @@ public class DamageOnLandComponent : Component, ILand
     {
         public override string Name => "DamageOnLand";
 
-        [DataField("damageType")]
-        private DamageType _damageType = DamageType.Blunt;
+        [DataField("damageType", required: true)]
+        private DamageTypePrototype _damageType = default!;
```

Comment:
Paul where are the prototype references

Reply:
![image](https://user-images.githubusercontent.com/7108570/129309012-88fd0cc7-db19-4c4e-a87d-5a6769711880.png)